include/addressatcommand.h

```diff
@@ -66,6 +66,28 @@ class AddressAtCommand : public NonKeyedCommand
         *            type.
         */
         virtual Command* clone();
+
+        /*!
+        *   \brief Returns host of database node
+        *   \param address The address of the database node
+        *   \returns The host of the database node
+        */
+        inline std::string parse_host(std::string address)
+        {
+            return address.substr(0, address.find(":"));
```

Comment:
It looks like ``substr`` can return ``out_of_range`` and ``bad_alloc`` exceptions.  Maybe we should account for those here (and below in parse_host) and either rethrow as is or as ``std::runtime_error`` so that when #16 is complete better handle failure with new custom error objects?  Also, do you think it is worth checking to make sure that ":" is in the string and not the first position (and if not) throw an error?